pkg/core/native/native_neo.go

```diff
@@ -200,6 +200,13 @@ func newNEO(cfg config.ProtocolConfiguration) *NEO {
 	md = newMethodAndPrice(n.unregisterCandidate, 1<<16, callflag.States)
 	n.AddMethod(md, desc)
 
+	desc = newDescriptor("onNEP17Payment", smartcontract.VoidType,
+		manifest.NewParameter("from", smartcontract.Hash160Type),
+		manifest.NewParameter("amount", smartcontract.IntegerType),
+		manifest.NewParameter("data", smartcontract.AnyType))
+	md = newMethodAndPrice(n.onNEP17Payment, 1<<15, callflag.States|callflag.AllowCall|callflag.AllowNotify, config.HFEchidna)
```

Comment:
> callflag.AllowCall

Is it for GAS contract call? I'm wondering because Notary's OnNEP17 payment has several "calls" to other contracts and only `callflag.States` flag, so may be it's worth fixing on Notary side.